docs/userguide/citing.rst

```diff
@@ -17,28 +17,40 @@ Example BibTeX entry:
 .. parsed-literal::
 
   @software{metpy,
-    author = {May, Ryan M. and Arms, Sean C. and Marsh, Patrick and Bruning, Eric and Leeman, John R.
-              and Goebbert, Kevin and Thielen, Jonathan E. and Bruick, Zachary S.},
+    author = {May, Ryan M. and Arms, Sean C. and Marsh, Patrick and Bruning, Eric and Leeman,
+              John R. and Goebbert, Kevin and Thielen, Jonathan E. and Bruick, Zachary S.},
     organization = {Unidata},
     title = {MetPy: A {Python} Package for Meteorological Data},
-    year = {2008 - |cite_year|},
+    year = {|cite_year|},
     version = {|cite_version|},
     doi = {10.5065/D6WW7G29},
     url = {https://github.com/Unidata/MetPy},
-    address = {Boulder, Colorado}
   }
 
 
 *****************
 AMS Journal Style
 *****************
 
+AMS suggested BibTeX entry:
+
+.. parsed-literal::
+
+  @misc{metpy,
+    author = {May, Ryan M. and Arms, Sean C. and Marsh, Patrick and Bruning, Eric and Leeman,
+              John R. and Goebbert, Kevin and Thielen, Jonathan E. and Bruick, Zachary S.},
+    publisher = {Unidata},
+    title = {MetPy: A {Python} Package for Meteorological Data},
+    year = {|cite_year|},
+    doi = {10.5065/D6WW7G29},
+    url = {https://github.com/Unidata/MetPy},
```

Comment:
I think what we want is 
``NOTE    = {accessed 18 December 2020, \url{https://github.com/Unidata/MetPy}}``

instead of just the URL.

Reply:
I wasn't sure if the "accessed <date>" was actually helpful for the software or not. Since AMS didn't list it for that, I was ok letting it go. I'm ok restoring it too.

Reply:
The only reason why I think it might be important to keep is that the software does change (bug fixes) that might be important, especially around calculations. But I guess that is also captured in the version number. Since AMS has no official listing for software on the website (just datasets), let's forgo the accessed piece at this time, but keep the URL in, which I think would still work best in the "NOTE" field.